packages/medusa/src/services/payment-provider.ts

```diff
@@ -368,11 +368,14 @@ export default class PaymentProviderService extends TransactionBaseService {
     }
   }
 
-  async createPayment(
-    cart: Cart & { payment_session: PaymentSession }
-  ): Promise<Payment> {
+  async createPayment(data: {
+    cartId: string
```

Comment:
todo: sorry my previous comment was off - should be snake cased
```suggestion
    cart_id: string
```